assignments/callbacks.js

```diff
@@ -1,34 +1,92 @@
-const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
-
+const items = ['Pencil', 'Gum', 'Notebook', 'yo-yo', 'Gum'];
+//===========================================
 function firstItem(arr, cb) {
   // firstItem passes the first item of the given array to the callback function.
+  return cb(arr);
+}
+function itemFirst(arr) {
+  return arr[0];
 }
+console.log(firstItem(items, itemFirst));
```

Comment:
I see what you are doing here, using the itemFirst as your cb function however there was a less convoluted way to handle this.
```
fn firstItem(arr, cb) {
cb(arr[0]);
}
````
This serves the same purpose and is a little more explicit that we are using a cb function. When we invoke the firstItem function, the first item in arr will be passed to the callback function. The wording of these questions makes it sound much more complicated than in actually is. This goes for all of the following problems. I'm fairly sure that you understand the concept, just want you to see that you could have written it all much simpler for this line and all the following instances where you plugged it into a console.log.